README.md

```diff
@@ -88,7 +101,7 @@ Run mypy checks:
 
 Run pep8 checks:
 
-    pep8 tracim
+    pep8 tracim --exclude 'tracim/env'
```

Comment:
according to the documentation above, I believe the virtualenv is in `tracim_backend/env`, not in `tracim_backend/tracim/env`